### ISSUE TITLE
LODs don't work if part of the scene

### ISSUE THREAD

**Issue by mesealexan** [mozilla/hubs#6372]
**Description**
When I'm adding a GLB file that contains LODs it doesn't switch between the LODs. If the glb file is added in Spoke it doesn't work but if it's uploaded on an existing scene it works.
Here's a video showing the isseu:
One teapot is part of the scene (the one that is not working) and the other is up
loaded on the spot (the one that is working) https://imgur.com/a/tLf6r8G

**To Reproduce**
1. Go to subscription instance
2. Add a glb file with LODs setup: 
[test.zip](https://github.com/mozilla/hubs/files/13377382/test.zip)

3. Publish the scene
4. The LODs don't work
5. If you drag and drop the same glb file in the scene it will work

**Expected behavior**
The LODs file should work if the GLB file is part of the scene

**Screenshots**

**Hardware** 
 - Device: [e.g. Desktop, phone, VR headset]
 - OS: [e.g. Windows, iOS, Linux]
 - Browser: [e.g. Firefox]

**Additional context**

**Comment by mikemorran** [comment]
@mesealexan thanks for reporting this issue. If I am correct, the LODs set using Blender utilize a different add-on from the blender add on maintained by @takahirox: https://github.com/takahirox/glTF-Blender-IO-MSFT-lod

I think there is a discussion to be had about whether adding LODs at the scene-level are best accomplished in Spoke, the Hubs Blender Add-On, or both. @keianhzo @Exairnous any thoughts on this?

**Comment by Exairnous** [comment]
@mikemorran Yes, this is using the add-on maintained by Takahiro.  My thoughts are that adding LODs should be done at a Blender level, but assets with LODs should be able to be used in Spoke.  Currently to use LODs in a scene you need to create the entire scene in Blender and use Spoke's `Import From Blender` option; if you try to publish a scene through the Spoke editor, any assets with LODs will have the LODs stripped out and I think this not intuitive and is a bug to be fixed.

As discussed at the Friday meetup, since this is a bug in Spoke when publishing, and not related to the Hubs client, could you please transfer this issue to the Spoke repo?
